test/parallel/test-crypto-pbkdf2.js

```diff
@@ -61,7 +56,7 @@ function ondone(err, key) {
 
 // Error path should not leak memory (check with valgrind).
```

Comment:
Is this comment accurate for the test immediately below it? It's not obvious to me.

Reply:
I don't know, to be honest. Can look at that separately as it doesn't see to be relevant to this change